ufersa/plataforma-sabia
# Update footer

## Summary


Resolves #711 

:exclamation: Needs review at the [blog repository](https://github.com/ufersa/plataforma-sabia-blog-web/pull/6) as well. :exclamation: 

## Relevant technical choices


Removed old footer and kept one for both the platform and LP/about.

I have also modified the about header, which now has a link to the platform (início) and updated the user styles to fit all of the links better.

## QA Steps


1. Open any page of the platform and verify that the footer now looks like this:
![image](https://user-images.githubusercontent.com/10481028/102690990-0b231a00-41e8-11eb-9496-1d52874d3825.png)
2. Open the about page and verify that the footer looks the same as above.
3. Open the about page and verify that the header now looks like this:
![image](https://user-images.githubusercontent.com/10481028/102691042-6e14b100-41e8-11eb-941b-df1e446e1097.png)


## Checklist
